src/spotted/config/yaml/settings.yaml

```diff
@@ -19,5 +19,7 @@ post:
   delete_anonymous_comments: true
   reject_after_autoreply: true
   autoreplies_per_page: 6
+  blacklist_messages: ["myspamword1", "myspamword2"]
+
```

Comment:
Should those words be in the default configuration?

Reply:
replaced